.changeset/config.json

```
@@ -1,6 +1,6 @@
 {
   "$schema": "https://unpkg.com/@changesets/config@0.2.1/schema.json",
-  "changelog": ["@changesets/changelog-github", { "repo": "keystonejs/keystone" }],
+  "changelog": ["@changesets/changelog-github", { "repo": "keystone-next/keystone" }],
```

```suggestion
  "changelog": ["@changesets/changelog-github", { "repo": "keystonejs/keystone" }],
```

This refers to the repo itself.